### PR TITLE
Add person crud

### PR DESCRIPTION
**Pull request by brentvatne**
Mostly in place, missing a confirmation button for destroy.

### CODE REVIEW

File: app/controllers/people_controller.rb [hunk 1]
```diff
@@ -1,13 +1,75 @@
...
+          head :no_content
+        end
+      else
+        format.html do
+          flash[:alert] = "Unable to update #{@person.github_nickname}"
+          render :action => "edit"
+        end
+        format.json do
+          render :json => @person.errors, :status => :unprocessable_entity
+        end
```
Comment (jordanbyron) [inline app/controllers/people_controller.rb]: We don't need these format.json blocks. Right now we only need to render json for the `show` action. All the others can be removed.

**Comment by jordanbyron** [comment]
@brentvatne would you mind re-structuring your test support files to follow the same design as what we did in Liskov? The general idea is the same as yours, but I just moved the specs into their own file:

[test/support/spec.rb](https://github.com/mendicant-university/liskov/blob/master/test/support/spec.rb)
[test/test_helper.rb](https://github.com/mendicant-university/liskov/blob/master/test/test_helper.rb)

Let me know how you're doing with your workload. If you're really busy maybe @semmons99 or I could make those changes. Thanks :smile:

**Comment by semmons99** [comment]
@jordanbyron can this be merged into clubhouse? I'd like to work on the person search in a fresh branch if possible.

**Comment by brentvatne** [comment]
I think it's fine to merge in, afterwards I'll create a separate branch to make a nicer destroy button.

**Comment by jordanbyron** [comment]
:sparkles: Nice works @semmons99 & @brentvatne! I made a few very minor changes to your code. Check them out and let me know if you have any questions about what I did: bf03d33

**Pull request merged**